src/inference/src/dev/converter_utils.cpp

```diff
@@ -696,8 +698,8 @@ class IAsyncInferRequestWrapper : public ov::IAsyncInferRequest {
                 info.status = ov::ProfilingInfo::Status::EXECUTED;
                 break;
             }
-            info.real_time = std::chrono::microseconds{ieInfo.realTime_uSec};
-            info.cpu_time = std::chrono::microseconds{ieInfo.cpu_uSec};
+            info.real_time = std::chrono::microseconds{static_cast<long long>(us2ms * ieInfo.realTime_uSec)};
+            info.cpu_time = std::chrono::microseconds{static_cast<long long>(us2ms * ieInfo.cpu_uSec)};
```

Comment:
This also reads strange: You convert `ieInfo.cpu_uSec` to milliseconds using `us2ms`, but store the result to `std::chrono::microseconds`.